src/extension.ts

```diff
@@ -421,14 +421,14 @@ function maybeRunKubernetesCommandForActiveWindow(command) {
                         vscode.window.showErrorMessage("Save failed.");
                         return;
                     }
-                    kubectl.invoke(`${command} "${editor.document.fileName}"`);
+                    kubectl.invokeInTerminal(`${command} "${editor.document.fileName}"`);
```

Comment:
This looks like it will affect Create, Apply and Diff as well as Get?  Per Kylie's notes, I think the former three are meant to _not_ go via the terminal, though Get is.

Reply:
Fixed. Already undo the changes.